Makefile

```diff
@@ -125,8 +125,32 @@ TESTPKGS ?= github.com/siderolabs/talos/...
 RELEASES ?= v1.8.4 v1.9.0
 SHORT_INTEGRATION_TEST ?=
 CUSTOM_CNI_URL ?=
+
 INSTALLER_ARCH ?= all
+INSTALLER_ONLY_PKGS ?= \
+    bash \
+    cpio \
+    efibootmgr \
+    kmod \
+    squashfs-tools \
+    xfsprogs \
+    xz \
+    zstd
+
+IMAGER_EXTRA_PKGS ?= \
```

Comment:
Maybe rename it to be somewhat relevant to imager (I guess this gets used there)?

Reply:
not sure how to make it more relevant, it's called `IMAGER_EXTRA_PKGS` :)

Reply:
nvm